.github/ISSUE_TEMPLATE/-bug-report-.md

```diff
@@ -0,0 +1,30 @@
+---
+name: "[Bug Report]"
+about: For noting problems or unexpected behavior
+
+---
+
+**Bug Description**
+Please enter a clear and concise description of what the bug is.
+
+**Reproduction Steps**
+Enter steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+Give a clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Client logs:**
```

Comment:
"In Chrome, javascript logs can be found via View -> Developer -> JavaScript Console."